src/instantiate.ts

```diff
@@ -214,7 +214,7 @@ export async function loadAllofExtension(context: vscode.ExtensionContext) {
       quickPick.onDidHide(() => quickPick.dispose());
       quickPick.show();
     }),
-    vscode.commands.registerCommand(`code-for-ibmi.runAction`, async (node) => {
+    vscode.commands.registerCommand(`code-for-ibmi.runAction`, async (node: any, group?: any, action?: Action, method?: DeploymentMethod) => {
```

Comment:
`group` seems to be unused here.
```suggestion
    vscode.commands.registerCommand(`code-for-ibmi.runAction`, async (node: any, action?: Action, method?: DeploymentMethod) => {
```

Reply:
Although this parameter is unused, we still need it because when `code-for-ibmi.runAction` is invoked from the editor's context menu, VS Code passes in this parameter (it has to do with the editor tab group).

![image](https://github.com/halcyon-tech/vscode-ibmi/assets/32170854/089cbaa4-e9a1-48ae-995c-13ef005dc682)

Reply:
Ha right, good catch 😅